doc/nrf/ug_zigbee_platform_design.rst

```diff
@@ -0,0 +1,99 @@
+
+System-on-Chip designs
+**********************
+
+This single-chip solution has the combined RFIC (the IEEE 802.15.4 in case of Zigbee) and processor. The ZIgbee stack and the application layer run on the local processor.
+
+Single-chip, single protocol (SoC)
+==================================
+
+In this design, the application layer and the stack run on the same processor. The application uses the ZBOSS APIs directly.
+
+This is the SoC design most commonly used for end devices and routers.
+
+This design has the following advantages:
+
+* Lowest cost.
+* Lowest power consumption.
+* Lowest complexity.
+
+It also has the following disadvantages:
+
+* For some uses, the nRF528xx MCU can be too slow. For example, when the application would like to do complex data processing.
+* The application and the network share Flash and RAM space, which can limit the application functionality.
+* Dual-bank DFU or an external flash is needed to update the firmware.
+
+.. figure:: /images/zigbee_platform_design_soc.svg
+   :alt: Zigbee-only architecture.
+
+For more information, see :ref:`zigbee_samples`.
+
+Single-chip, multiprotocol (SoC)
+================================
+
+With the nRF52 devices supporting multiple wireless technologies, including IEEE 802.15.4 and Bluetooth Low Energy (BLE), the application layer and the Zigbee and BLE stack run on the same processor.
+
+This design has the following advantages:
+
+* It leverages the benefits of highly integrated SoC, resulting in the lowest cost and the lowest power consumption.
+* It allows to run Zigbee and BLE simultanously on a single chip, which reduces the overall BOM cost.
+
+It also has the following disadvantages:
+
+* BLE activity can degradate the connectivity on Zigbee if not implemented with efficiency in mind.
+
+.. figure:: /images/zigbee_platform_design_multi.svg
+   :alt: Multiprotocol Zigbee and BLE architecture.
+
+For more information, see :ref:`ug_multiprotocol_support` and :ref:`zigbee_light_switch_sample_nus`.
+
+.. _ug_zigbee_platform_design_ncp:
+
+Co-processor designs
+********************
+
+The co-processor designs can be used when a more powerful processor (host) that does not have a suitable radio interface. In these designs, the host interacts with the Zigbee network through Network Co-Processor (NCP).
+
+Split-stack co-processor design
+===============================
+
+In this design, the host processor runs only the Zigbee application layer (ZCL). The NCP runs the Zigbee MAC and the Zigbee Pro network layer as well as provides commands to execute BDB commissioning primitives. The host processor communicates with the NCP through a serial interface (typically UART).
+
+.. figure:: /images/zigbee_platform_design_ncp.svg
+   :alt: Split Zigbee architecture.
+
+All components, relevant for NCP host applcation are available in the dedicated `NCP SDK repository on GitHub <https://github.com/nordicsemiconductor>`_:
+
+* ZBOSS NCP protocol specification.
```

Comment:
Are you sure this is going to be published?

Reply:
No, I'm not. In fact - I'm not sure about any of those points and I was planning to update this list once the first SDK for NCP host will be released.

Reply:
Reduced to pieces, that are necessary to make a release.